src/index.html

```diff
@@ -11,7 +11,172 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form 
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+    method="post"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <label for="surname" class="labels">
+          Surname:
+          <input 
+            id="surname"
+            type="text" 
+            name="surname"
+            class="field"
+          >
+        </label>
+        <label for="name" class="labels">
+          Name:
+          <input 
+            id="name"
+            type="text" 
+            name="name"
+            class="field"
+          >
+        </label>
+        <label for="age" class="labels">
+          How old are You?
+          <input 
+            id="age"
+            min="1"
+            max="100"
+            value="12"
+            type="number" 
+            name="age" 
+            size="4"
+            class="field"
+          >
+        </label>
+        <label for="date" class="labels">
+          Full date of birth:
+          <input 
+            id="date"
+            type="date" 
+            name="dateOfBirth" 
+            min="1970"
+            class="field"
+          >
+        </label>
+        <label for="agreement" class="labels">
+          I accept the term of the agreement
+          <input 
+            id="agreement"
+            type="checkbox" 
+            name="agreement" 
+            required
+          >
+        </label>
+        
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <label for="email" class="labels">
+          E-mail:
+          <input 
+            id="email"
+            type="email" 
+            name="email"
+            class="field"
+            placeholder="email@example.com"
+          ><br>
+        </label>
+        <label for="password" class="labels">
+          Password:
+          <input 
+            id="password"
+            type="password" 
+            name="password"
+            minlength="6"
+            maxlength="36"
+          ><br>
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="fieldSelect">
+          <label class="labels">
+          Do you love cats?
+            <input 
+              type="radio"
+              name="pets"
+              value="yes"
+            >
+          Yes
+            <input 
+              type="radio"
+              name="pets"
+              value="no"
+            >
+          No <br>
+          </label>
```

Comment:
Because it's entirely wrapped in label, when I click anywhere in this row it selects "Yes" (even when I click on "No" as a word).
```suggestion
          <label class="labels">
            Do you love cats?
            <input 
              type="radio"
              name="pets"
              value="yes"
            >
            Yes
            <input 
              type="radio"
              name="pets"
              value="no"
            >
            No 
            <br>
          </label>
```

Reply:
Fixed